demos/button.html

```diff
@@ -306,6 +306,20 @@ <h1 class="mdc-typography--display2">CSS Only</h1>
             </button>
           </div>
         </fieldset>
+
+        <fieldset>
+          <legend class="mdc-typography--title">Different Color Icons/Text</legend>
+          <div>
+            <button class="mdc-button mdc-button--unelevated demo-text-color" data-demo-no-js>
+              <i class="material-icons mdc-button__icon">favorite</i>
+              Big Corner Radius
```

Comment:
Update the labels on these buttons; these labels were copied from the previous two examples but don't make sense here.